### PR TITLE
Added DigitalOcean as Cloud Provider

### PR DESCRIPTION
**Pull request by bobbyiliev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,11 @@ Table of Contents
     * .tech Domain Name - Free to use for one year
     * Full, detailed list - https://www.alibabacloud.com/campaign/free-trial
 
+
+  * [DigitalOcean](https://www.digitalocean.com/)
+    * [GitHub Student Developer Pack for free $50 DigitalOcean Credit](https://www.digitalocean.com/github-students/)
```
Comment (ripienaar) [inline README.md:137]: this is probably a bt low to bother with tbh, $50 doesnt get you far not over a year

File: README.md [hunk 2]
```diff
@@ -132,6 +132,11 @@ Table of Contents
...
+    * [DigitalOcean Open source sponsorships](https://www.digitalocean.com/open-source/)
```
Comment (ripienaar) [inline README.md:138]: I dont know how they decide who to sponsor, but its not a thing thats generally available to OSS projects.

Comment (bobbyiliev) [reply README.md:138]: Hi @ripienaar,

I've seen a lot of open source projects being sponsored by DigitalOcean. I guess that it depends on the project and the credit requirements, but speaking for experience, as long as you have an open-source project and submit an inquiry, you will always get a reply.

Comment (ripienaar) [reply README.md:138]: Appreciate that, but I have a requirement to make it easy to verify the offers on this list is legit and meet the stated requirement.  With a 'please mail us for details' offer, that's impossible for me and simply too much time to devote.  Imagine that * 500 services.

Comment (maple3142) [reply README.md:138]: > Hi @ripienaar,
> 
> I've seen a lot of open source projects being sponsored by DigitalOcean. I guess that it depends on the project and the credit requirements, but speaking for experience, as long as you have an open-source project and submit an inquiry, you will always get a reply.

But getting a reply != accepted, so I don't think it is really fit this list. I guess that it is only for medium to big size projects, please correct me if it isn't.

Comment (ripienaar) [reply README.md:138]: > 
> But getting a reply != accepted, so I don't think it is really fit this list. I guess that it is only for medium to big size projects, please correct me if it isn't.

Thats the impression I get too

Comment (maple3142) [reply README.md:138]: Okay, I seems to be wrong about it is only for medium to big size project. I found [this](https://docs.google.com/document/d/1P1ZqpxgKtiNibtWs53dLeCBmfp47Nqi6QlRg74lbsAc/edit#), and it said only 100 stars are needed.

Comment (ripienaar) [reply README.md:138]: Thank you @maple3142

**Pull request closed without merging**